CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 * Remove unnecessary JSDoc type annotations from generated `.d.ts` files
   [#4187](https://github.com/rustwasm/wasm-bindgen/pull/4187)
 
+* Optional parameters are now typed as `T | undefined | null` to reflect the actual JS behavior.
```

Comment:
```suggestion
* Optional parameters are now typed as `T | null` to reflect the actual JS behavior.
```

Reply:
This would be incorrect. We may *write* an optional parameter as `arg?: T | null` since `undefined` is implied by `?`, but the type of `arg` *is* `T | undefined | null`.

Reply:
Ah, right!